src/components/graph-controls.js

```diff
@@ -102,15 +102,15 @@ class GraphControls extends Component {
   }
 
   // Modify current zoom of graph-view
-  zoom(e){
+  zoom = (e) => {
```

Comment:
The changes in this file are automatic formatting due to slight changes while finding the bug. We also didn't need the bind call, so I removed that.